contracts/p1/RToken.sol

```diff
@@ -113,6 +113,9 @@ contract RTokenP1 is ComponentP1, ERC20PermitUpgradeable, IRToken {
     /// @param recipient The address to receive the issued RTokens
     /// @param amount {qRTok} The quantity of RToken to issue
     /// @custom:interaction
+    // untestable:
+    //      `else` branch of `exchangeRateIsValidAfter` (ie. revert)
+    //       It is not possible to change the BU exchange rate through issue
```

Comment:
Nit: It is possible to increase the BU exchange rate through issue, but _only_ when the exchange rate is less than FIX_ONE.

Reply:
So what do you suggest as comment? Something like:   `BU exchange can increase, but only when less than FIX_ONE.` Would be OK?

Reply:
Sounds good to me. And it cannot decrease.